app/assets/stylesheets/profile.scss

```diff
@@ -0,0 +1,114 @@
+$black: #000000;
```

Comment:
What you can do with these colors, is to create a new `colors.scss` that only contains these colors. Then, in `application.scss`, import these colors after `bootstrap` and before everything else.